.github/CODEOWNERS

```
@@ -0,0 +1,30 @@
+# Default reviewers
+* @allyhawkins @jashapiro @jaclyn-taroni @sjspielman
+
+# CODEOWNER owner
+/.github/CODEOWNERS @jaclyn-taroni
+
+# Primary reviewers for docs and licenses changes
+/docs/ @dvenprasad @sjspielman
+/CONTRIBUTING.md @dvenprasad @sjspielman
+/README.md @dvenprasad @sjspielman
+/mkdocs.yml @dvenprasad @sjspielman
+/LICENSE.md @jaclyn-taroni
+/SECURITY.md @jaclyn-taroni
+/code-of-conduct.md @jaclyn-taroni
```

I should be assigned to anything that changes _any_ policy:
```suggestion
/code-of-conduct.md @jaclyn-taroni
/docs/policies/ @jaclyn-taroni 
```